flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -261,7 +266,7 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
 
         do {
             let envelope = try parseJsonEnvelope(event)
-            SentrySDK.currentHub().capture(envelope: envelope)
+            SentrySDK.capture(envelope: envelope)
```

Comment:
@philipphofmann Could you please explain to me how I would make this available?
You mentioned some way [here](https://github.com/getsentry/develop/issues/248#issuecomment-765206789) but I'm not quite sure how I would do that.

Reply:
Sorry for the late response. You can create a category which makes this function public to you. Create a file named `SentrySDK+FlutterPrivate.h` with something like 
```
#import "SentrySDK.h"

NS_ASSUME_NONNULL_BEGIN

@interface SentrySDK (FlutterPrivate)

+ (void)captureEnvelope:(SentryEnvelope *)envelope;

@end

NS_ASSUME_NONNULL_END
```

Let me know if this works.

Reply:
Thanks! Do I need to change the import in the Swift file for that to work?

Reply:
if I understood correctly, you just need to create a new file as suggested and import this new interface to be used in the current SentryFlutterPluginApple

Reply:
See https://github.com/getsentry/sentry-cocoa/blob/master/Tests/SentryTests/SentrySDK%2BTests.h
then https://github.com/getsentry/sentry-cocoa/blob/master/Tests/SentryTests/SentrySDKTests.swift#L295-L303

`capture(envelope)` is the alias for `captureEnvelope` in swift

Reply:
can you confirm @philipphofmann ? thanks

Reply:
There's no need to change any imports. Thanks again for your help 👍